PluginInterface/swig.cs

```diff
@@ -2069,7 +2069,7 @@ namespace FreeSWITCH {
 public class SWIGTYPE_p_switch_event_t {
   private global::System.Runtime.InteropServices.HandleRef swigCPtr;
 
-  internal SWIGTYPE_p_switch_event_t(global::System.IntPtr cPtr, bool futureUse) {
+  public SWIGTYPE_p_switch_event_t(global::System.IntPtr cPtr, bool futureUse) {
     swigCPtr = new global::System.Runtime.InteropServices.HandleRef(this, cPtr);
   }
 
```

Comment:
All the changes in this file appear to be made in this file, as this is generated by swig, we need to alter the swig.i to make these public upon regeneration.
I'm not exactly sure the correct format for the swig.i because this is for constructors, will have to look into this.
An example exists in the swig.i for a normal method, it should look a little something like this, with the obvious changes:
```%csmethodmodifiers CoreSession::originate "protected";```